bridges/snowbridge/pallets/outbound-queue-v2/src/benchmarking.rs

```diff
@@ -67,14 +99,48 @@ mod benchmarks {
 		Ok(())
 	}
 
-	/// Benchmark for producing commitment for a single message
+	/// Benchmark for producing commitment for a single message, used to estimate the delivery
+	/// cost. The assumption is that cost of commit a single message is even higher than the average
+	/// cost of commit all messages.
 	#[benchmark]
 	fn commit_single() -> Result<(), BenchmarkError> {
```

Comment:
Is `commit_single` still used or can we remove it?

Reply:
It's still used for estimating the delivery cost. The reason is explained in the comment above.